imports/plugins/included/notifications/client/components/notificationDropdown.js

```diff
@@ -93,7 +87,7 @@ class NotificationDropdown extends Component {
           })}
         </ul>
         <div className="dropdown-footer text-center">
-          <Link to={`${this.prefix}/notifications`} data-i18n="notifications.body.viewAll">View All</Link>
```

Comment:
Tested both scenarios and the `View All` link on the notification icon will direct you to the `Notifications Page`.